### PR TITLE
Add HH Economic Security (edu. attainment) Indicators from 2000 PUMS Data 

### PR DESCRIPTION
**Pull request by mbh329**
Few different things going on in this pull request including some reorganizing and renaming of files and what folders they are in, new code to handle the HH economic security indicators coming form 2000 census pums xlsx file dcp pop sent us, and a couple additions to the` aggregation_helpers.py `file that handles the reordering of the columns.

`pums_2000_economics.py`

Pretty straightforward. Follows the established conventions and logic to handle the xlsx files coming from dcp population. the data points (columns) of interest here are the educational attainment data points (less than high school, high school graduate or equivalent, etc) and the denominator (population 25 plus) which are the only data points coming from the 2000 pums for the hh econ security category. the main accessor now has a function that reorders the columns that builds off the work sasha had already done. It cycles through the 4 race crosstabs (anh, bnh, hsp, wnh) instead of the 5 (anh, bnh, hsp, onh, wnh) in the newer PUMS data.

`aggregation_helpers.py` 

Added a parameter to `order_aggregated_columns` that can handle the different categories. 

TODO:
Wanted to keep this pull req relatively small but there is still some refactoring that needs to be done. will make another pull req with these changes.

- Need to migrate all the column reordering to `aggregation_helpers.py`.  right now there are two files that handle the column reordering: `aggregation_helpers.py` and `clean_aggregated.py`. 
- I think we can safely assume that a lot of the xlsx coming from dcp pop can be processed using our established logic - need to move useful functions and dictionaries into a new utils file

### CODE REVIEW

**Comment by SashaWeinstein** [comment]
Awesome, really glad we are re-using code to reorder the columns and thinking about how to re-use code to ingest dcp pop data. 

I pulled down the code and realized that the a median age column for "onh" was added to demographics from 2000 census pums. I fixed that and also re-named a couple variables and added a docstring to identify the main accessor of the 2000 census pums demographics. 

Next I'll take a look at new `pums_2000_economics.py` file

**Review by SashaWeinstein — APPROVED** [review]
I left some comments on particular pieces of code that can be DRY'd out. Everything works on my machine though

Also internal review files shouldn't be in household economic security

**Comment by mbh329** [comment]
@SashaWeinstein What folder should the internal review files be in?

File: aggregate/PUMS/pums_2000_economics.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            df.loc[["BX", "BK", "MN", "QN", "SI"]]
+            .reset_index()
+            .rename(columns={"GeoID": "borough"})
+        )
+    else:
+        final = df.loc["3701":"4114"].reset_index().rename(columns={"GeoID": "puma"})
+        final["puma"] = final["puma"].apply(func=clean_PUMAs)
+
+    final.set_index(geography, inplace=True)
+    # TODO: migrate comment to comment
```
[the remark this answers is not in the document]
Comment (mbh329) [reply aggregate/PUMS/pums_2000_economics.py:106]: eh sorry I was adding comments for what could be put into the helper functions, not a helpful comment without more context - will remove

**Comment by SashaWeinstein** [comment]
oh shoot I got housing security and quality and household economic security mixed up, my bad. internal review review files are in the correct place

**Comment by mbh329** [comment]
I do that too

**Comment by mbh329** [comment]
@td928 Did you want to through this before i merge into dev?

**Pull request merged**